api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/NewClaimServiceTest.java

```diff
@@ -170,17 +175,74 @@ void shouldSaveClaimantWithClaimStatus(EligibilityStatus eligibilityStatus) {
     }
 
     @Test
-    void shouldSaveDuplicateClaimantForMatchingNino() {
+    void shouldUpdateClaimForMatchingNinoWhenEligible() {
         //given
-        Claimant claimant = aValidClaimant();
-        given(eligibilityAndEntitlementService.evaluateNewClaimant(any()))
-                .willReturn(EligibilityAndEntitlementDecision.buildWithStatus(DUPLICATE));
+        Claimant existingClaimant = aClaimantWithExpectedDeliveryDate(null);
+        LocalDate expectedDeliveryDate = LocalDate.now().plusMonths(6);
+        Claimant newClaimant = aClaimantWithExpectedDeliveryDate(expectedDeliveryDate);
+        Claim existingClaim = aClaimWithClaimant(existingClaimant);
+        UUID existingClaimId = UUID.randomUUID();
+        given(eligibilityAndEntitlementService.evaluateNewClaimant(any())).willReturn(EligibilityAndEntitlementDecision.builder()
```

Comment:
Can we use a test data factory for the `EligibilityAndEntitlementDecision` here?

Reply:
Sticking with builder due to number of fields being set.